README.md

```diff
@@ -218,3 +218,57 @@ removeFromFavorite: builder.mutation<void, number>({
   }
 })
 ```
+
+### Store Utils
+
+1. `createStoreInitializer` - this util is used for creating the application's `initStore`. It takes as arguments: `rootState`, `middlewares`(array), and `enhancers`(array). Example:
```

Comment:
Looks like we should use `rootReducer` instead of `rootState` here, or am I mistake?

Reply:
Yep, you are right :) My mistake. Thank you!